sdk/tables/Azure.Data.Tables/src/TableConstants.cs

```diff
@@ -37,7 +37,8 @@ internal static class PropertyNames
             public const string PartitionKey = "PartitionKey";
             public const string RowKey = "RowKey";
             public const string EtagOdata = "odata.etag";
-            public const string ETag = nameof(ITableEntity.ETag);
+            public const string OdataMetadata = "odata.metadata";
+            public const string ETag = "ETag";
```

Comment:
(extreme) nit:  A bit odd to have it `ETag` here and `EtagOdata` a short bit above.

Reply:
fixed 😄